ratom_api/settings/base.py

```diff
@@ -14,6 +14,7 @@
 
 BASE_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir))
 PROJECT_ROOT = os.path.abspath(os.path.join(BASE_DIR, os.pardir))
+SECRET_KEY = os.getenv('SECRET_KEY')
```

Comment:
This differs from ``DJANGO_SECRET_KEY`` in the deploy settings. Is that intentional? Or does this need to be in base?

Reply:
This can come out too. Part of me trying to figure out how to get passed the testing of the image.